lib/base64_url_decode.ts

```diff
@@ -10,7 +10,7 @@ function b64DecodeUnicode(str: string) {
   );
 }
 
-export default function(str: string) {
+export function base64UrlDecode(str: string) {
```

Comment:
Adding a function name here is also useful for stack traces, nice!

Reply:
Yeah, I personally try to avoid default exports. But I had no intention to change the code base too much, but it looks to be neccesary.